src/ontology/edits/oeo-physical.omn

```diff
@@ -505,6 +520,32 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1652"
         OEO_00000150
     
     
+ObjectProperty: OEO_00010238
```

Comment:
I think adding has energy participant here is an unintended error?

Reply:
Should be fixed now